tutorials/text_to_graph.ipynb

```diff
@@ -0,0 +1,192 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Text to Graph Transformation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Assume that we are working with some recommender model leveraging relationships among items, but for some dataset of interest the latter modality, i.e., item network, is not available, and instead we have access to another type of auxiliary information, let's say item textual descriptions. Rather than ignoring the available auxiliary information or changing to another algorithm integrating item texts, which could be costly, Cornac enables us to exploit the available textual information while using our model. This is possible thanks to Cornac's cross-modality transformation feature, which we will rely on in this tutorial.\n",
```

Comment:
I think the first sentence is too long. How about we break it down, for example:

Assume that we are working with some recommender model`s` leveraging relationships among items (item network). However, for some dataset`s` of interest, the required modality of the models is not available. Instead, we have access to another type of auxiliary information, let's say item textual descriptions.

Note that there are some typos of plural form.